CONTRIBUTING.md

```diff
@@ -25,11 +24,11 @@ gratitude and appreciation of everyone's time and work.
 Generally, the first steps are:
 
 * Fork this repository and clone it locally. Note that unlike the git default, our main branch is called `main`.
-* Get a working development environment (see the [README](https://github.com/MoveOnOrg/Spoke/#spoke) and [docs/](https://github.com/MoveOnOrg/Spoke/tree/main/docs))-- reach out through our communication channels (above) if you have issues.
+* Get a working development environment (see the [README](https://github.com/spokecommunity/Spoke/#spoke) and [docs/](https://github.com/spokecommunity/Spoke/tree/main/docs))-- reach out through our communication channels (above) if you have issues.
 
 ### Picking an issue
 
-* We mark issues that are good first issues with the [`good first issue` tag](https://github.com/MoveOnOrg/Spoke/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). Most of these are marked `easy`, but if you are a more experienced developer then some are not but it may be *easy for you*!
+* We mark issues that are good first issues with the [`good first issue` tag](https://github.com/spokecommunity/Spoke/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). Most of these are marked `easy`, but if you are a more experienced developer then some are not but it may be *easy for you*!
```

Comment:
Figuring it's okay to leave this in for now, even though there aren't actually any issues with this tag at the moment. We'll want to start creating some pretty soon.